app/models/employee.rb

```diff
@@ -25,11 +26,11 @@ class Employee < ActiveRecord::Base
   scope :team_employees, ->(user){joins(employee_teams: :employee).where(employee_teams: {team_id: user.employee_teams.pluck(:team_id)}).where.not(employee_teams: {employee_id: user.id}).distinct}
   scope :team_employees_projects_tasks, ->(user){joins(employee_teams: :employee).where(employee_teams: {team_id: user.employee_teams.pluck(:team_id)}).distinct}
 
-  def todays_attendance_of_employee(company)
+  def todays_attendance_of_employee
     # one employee should not have multiple attendances for one day
     @start_time = DateTime.now.change(hour: 10)
-    @end_time = DateTime.now.change(hour: 18)
-    company.attendances.find_by(employee_id: id, login_time: (@start_time..@end_time))
+    @end_time = DateTime.now.change(hour: 20)
+    self.company.attendances.find_by(employee_id: id, login_time: (@start_time..@end_time))
```

Comment:
i guess this can be done by
`self.attendances.find_by(login_time: (@start_time..@end_time))`

Reply:
done
also no need to use self in this model